hyprbars/barDeco.cpp

```diff
@@ -126,6 +129,39 @@ void CHyprBar::onMouseDown(SCallbackInfo& info, IPointer::SButtonEvent e) {
     m_bDragPending = true;
 }
 
+void CHyprBar::onTouchDown(SCallbackInfo& info) {
+    if (m_pWindow.lock() != g_pCompositor->m_pLastWindow.lock())
```

Comment:
both lock() redundant

Reply:
will change

Reply:
I don't think it is?

Reply:
sorry was testing because it brokey mb will fix now

Reply:
okay it is done

Reply:
and it is back again now that dragging exists